README.md

```diff
@@ -18,7 +17,19 @@ To get the source code of our SDKs and samples via **git** just type:
     git clone https://github.com/Azure/azure-mobile-services.git
     cd ./azure-mobile-services/
 
-## Change log
+## SDk Documentation references
+
+#### Azure App Service Mobile Apps
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v3)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v2)
+
+#### Mobile Services
+* [iOS](http://azure.github.io/azure-mobile-services/ios/v2)
+* [Managed](http://azure.github.io/azure-mobile-services/managed/v1)
+* [Android](http://azure.github.io/azure-mobile-services/android/v2)
+* [Javascript](http://azure.github.io/azure-mobile-services/javascript/v1)
+
+## SDK Change log
```

Comment:
Also, if we are going to spell out pointers to Apps & services in the readme, we may want to do it for teh change logs as well.  Talk to @lindydonna